lib/service/cfg.go

```diff
@@ -359,7 +359,7 @@ func (c ProxyConfig) KubeAddr() (string, error) {
 	if len(c.PublicAddrs) > 0 {
 		host = c.PublicAddrs[0].Host()
 	}
-	return fmt.Sprintf("https://%s:%d", host, c.Kube.ListenAddr.Port(defaults.KubeProxyListenPort)), nil
+	return fmt.Sprintf("https://%s:%d", host, c.Kube.ListenAddr.Port(defaults.KubeListenPort)), nil
```

Comment:
Hrm, what about the following, it will probably support IPv6.

```
u := url.URL{
  Scheme: "https",
  Host: host,
  Port: c.Kube.ListenAddr.Port(defaults.KubeListenPort),
}
return u.String()
```

Reply:
done